tests/manager/feed/test_opds_acquisition_feed.py

```diff
@@ -929,15 +913,19 @@ def facet_link(cls, url, facet_title, group_title, selected, is_default):
         assert not is_default
 
     def test_active_loans_for_with_holds(
-        self, db: DatabaseTransactionFixture, patch_url_for: PatchedUrlFor
+        self,
+        db: DatabaseTransactionFixture,
+        patch_url_for: PatchedUrlFor,
+        services_fixture: ServicesFixture,
     ):
         patron = db.patron()
         work = db.work(with_license_pool=True)
         hold, _ = work.active_license_pool().on_hold_to(patron)
 
-        feed = OPDSAcquisitionFeed.active_loans_for(
-            None, patron, LibraryAnnotator(None, None, db.default_library())
-        )
+        with services_fixture.wired():
+            feed = OPDSAcquisitionFeed.active_loans_for(
+                None, patron, LibraryAnnotator(None, None, db.default_library())
+            )
```

Comment:
This change is needed because without a wired service container we get an exception (and have been for awhile) that was being swallowed by the except clause I removed.